modules/cli11/2.4.2/MODULE.bazel

```diff
@@ -0,0 +1,7 @@
+module(
+    name = "CLI11",
```

Comment:
Valid module names must begin with a lowercase letter. Upstream chose `cli11`, so let's stick to that.